_pages/community-service/german.html

```diff
@@ -1,7 +1,6 @@
 ---
 layout: default
 title: Community Service
-permalink: /community-service/german
```

Comment:
It’s to prevent the pages from being accessed on the web, since they’re out of date. I thought removing permalinks would be the best way to archive a copy of the page and make it easy to update and make live again in the future.

Reply:
Hmmm. I get what you're saying but we should at least add some sort of warning so a future maintainer can be aware of how to fix this, as it's a highly non-obvious solution. Alternatively can we just remove all links to the page and leave it sitting there? Up to you.

Reply:
Sounds good. I just added permalinks back to all of the archived pages. There shouldn't be links to them anywhere on the site.